context/app/api/vitessce_confs/base_confs.py

```diff
@@ -237,7 +237,7 @@ def build_vitessce_conf(self):
                 if path not in file_paths_found:
                     message = f'SPRM file {path} with uuid "{self._uuid}" not found as expected.'
                     if not self._is_mock:
-                        current_app.logger.info(message)
```

Comment:
Could get rid of this logger call all together since the error is thrown anyway and the decorator handles it